tests/robot-tests/tests/admin_and_public/bau/prelease_visibility.robot

```diff
@@ -0,0 +1,223 @@
+    user enters text into element  id:releaseStatusForm-publishScheduled-day    ${day}
+    user enters text into element  id:releaseStatusForm-publishScheduled-month  ${month}
+    user enters text into element  id:releaseStatusForm-publishScheduled-year   ${year}
+    user enters text into element  id:releaseStatusForm-nextReleaseDate-month   1
+    user enters text into element  id:releaseStatusForm-nextReleaseDate-year    2001
+    user clicks button   Update status
+
+    user waits until h2 is visible  Sign off
+    user checks summary list contains  Current status  Approved
+    user checks summary list contains  Scheduled release  ${day} ${month_word} ${year}
+    user checks summary list contains  Next release expected  January 2001
+    user waits for release process status to be  Scheduled  90
+
+Navigate to prerelease page
+    [Tags]  HappyPath
+    ${current_url}=  get location
+    ${RELEASE_URL}=  remove substring from right of string  ${current_url}  /status
+    set suite variable  ${RELEASE_URL}
+    user goes to url   ${RELEASE_URL}/prerelease
+
+Validate prerelease has not started
+    [Tags]  HappyPath
+    user waits until h1 is visible  Pre-release access is not yet available
+    user checks breadcrumb count should be   2
+    user checks nth breadcrumb contains   1    Home
+    user checks nth breadcrumb contains   2    Pre-release access
+
+    ${day}=         get current datetime  %d   1
+    ${month}=       get current datetime  %m   1
+    ${year}=        get current datetime  %Y   1
+    ${time_start}=  format uk to local datetime  ${year}-${month}-${day}T00:00:00  %-d %B %Y at %H:%M
+    ${time_end}=    format uk to local datetime  ${year}-${month}-${day}T23:59:00  %-d %B %Y at %H:%M
+    user checks page contains   Pre-release access will be available from ${time_start} until ${time_end}.
+
+Go to prerelease access page
+    [Tags]  HappyPath
+    user goes to url   ${RELEASE_URL}/prerelease-access
+    user waits until h2 is visible  Manage pre-release user access
+
+Go to Public release page and ensure release isn't visible
+    [Tags]  HappyPath
+    user goes to url  %{PUBLIC_URL}
+    user goes to url  ${PUBLIC_RELEASE_LINK}
+
+Go to Table Tool page
+    [Tags]  HappyPath
+    environment variable should be set  PUBLIC_URL
+    user goes to url  %{PUBLIC_URL}/data-tables
+    user waits until h1 is visible  Create your own tables online
+
+Check scheduled release isn't visible
+    [Tags]  HappyPath
+    environment variable should be set  TEST_THEME_NAME
+    user opens details dropdown    %{TEST_THEME_NAME}
+    user checks page does not contain  UI test topic %{RUN_IDENTIFIER}
+
+Go to release URL and check release isn't visible
+    user goes to url  ${PUBLIC_RELEASE_LINK}
+    # NOTE: Locally this will fail due to next showing the default dev 404 page
+    user checks page contains element  //*[@data-testid="page-title Page not found" and text()="Page not found"]
```

Comment:
Same as above - I'd use `user checks page contains   Page not found` and add another check that the release name doesn't appear on the page.